src/models/slices/vertical.ts

```diff
@@ -1,11 +1,34 @@
-import { VerticalSearchResponse, AutocompleteResponse, VerticalResults } from '@yext/answers-core';
+import {
+  AppliedQueryFilter,
+  AutocompleteResponse,
+  DisplayableFacet,
+  Result,
+  SearchIntent,
+  SortBy,
+  Source,
+  VerticalResults
+} from '@yext/answers-core';
+
+export interface AllResultsForVertical {
+  facets: DisplayableFacet[],
+  results: Result[],
+  resultsCount: number,
+  searchIntents: SearchIntent[]
+}
+
 export interface VerticalSearchState {
-  alternativeVerticals?: VerticalResults[]
-  key?: string,
+  allResultsForVertical?: AllResultsForVertical,
+  alternativeVerticals?: VerticalResults[],
```

Comment:
I'm wondering if we should group these two under an optional attribute called `noResults?` or something. The current state design doesn't make it clear when we can or can't expect this data. I suppose we could also solve that problem with documentation.

Reply:
Currently, client code trying to use alternativeVerticals or allResultsForVertical would look something like: `if (state.vertical.resultsCount === 0 && state.vertical.alternativeVerticals)`, which reads well to me. However I don't have a strong preference here. I can ping Rose

Reply:
Rose liked the noResults prop so i'll update!